### ISSUE TITLE
Don't trigger WeakDelegateRule if it is computed property

### ISSUE THREAD

**Issue by mpsnp** [realm/SwiftLint#1089]
In my project i have a factory of AppDelegates, in order to simplify resolving, i've added computed properties, which look like this:

```swift
var normalLaunchAppDelegate: NormalLaunchAppDelegate {
    return resolve()
}
```

I'm pretty sure that in this case WeakDelegateRule shouldn't trigger.

**Comment by marcelofabri** [comment]
This should be an easy enhancement. @mpsnp, would you like to give it a shot? 😄
